### DISCUSSION TITLE
Parameter q

### DISCUSSION THREAD

**Discussion by dbogatic** [dcajasn/Riskfolio-Lib#34] [Q&A] [answered]
Can you explain to me what is d in

`port.assets_stats(method_mu=method_mu, method_cov=method_cov, d=0.94)`

Thank you.

Dragan

**Comment by dcajasn** [comment, marked as answer]
All parameters are well explained in docs, but this d is for ewma estimators.

**Reply by dbogatic** [reply]
Yes, I did look over all of them and this one was missing. Could you explain what it does? I know what ewma is, just want to make sure I understant the input. If it is there and I missed it , I will go back and read again. Thank  you for your help!

**Reply by dcajasn** [reply]
It's just the alpha factor in pandas ewm https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.ewm.html. You must check parameters estimation documentation. In that section, I explain all input estimation techniques available in Riskfolio-Lib (ewma, shrinkage, worst case, black litterman, etc)

**Reply by dbogatic** [reply]
d (scalar) – The smoothing factor of ewma methods. The default is 0.94.

Found it...thank you!
